lib/client.js

```diff
@@ -172,6 +172,31 @@ var GiantSwarm = (function () {
     instanceStats: function(organizationName, instanceId, successCallback, errorCallback) {
       var uri = '/v1/org/' + organizationName + '/instance/' + instanceId + '/stats';
       this._simple_data_get_request(uri, successCallback, errorCallback);
+    },
+
+    /**
+     * Create a websocket connection to stream logs from instances
+     */
+    streamLogs: function(organizationName, instanceIds, messageCallback, createCallback, errorCallback) {
+      var uri = '/v1/org/' + organizationName + '/stream/logs';
+      request
+        .post(apiEndpoint + uri)
+        .set('Authorization', 'giantswarm ' + authToken)
```

Comment:
The authorization header (and probably a few others) will have to be set in all API calls.
So I suggest to group them in a separate function.